README.md

```diff
@@ -225,8 +247,16 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License <a name="license"></a>
 
+
+This project is [MIT]() licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+=======
 This project is [MIT](https://github.com/maherramadan78/leaderboard) licensed.
```

Comment:
- [ ] It would be better if we address the issue about the broken links; In my opinion, an example of a broken link is our current `This project is [MIT](https://github.com/maherramadan78/leaderboard) licensed.`. When a user clicks on this link, he is expected to be taken to the MIT license page but unfortunately for us we are redirected to a `404` page. In this case, the link is broken I guess